rest/taskana-rest-spring-example-common/src/test/resources/taskana.properties

```diff
@@ -12,3 +12,4 @@ taskana.jobs.batchSize=50
 taskana.jobs.cleanup.runEvery=P1D
 taskana.jobs.cleanup.firstRunAt=2018-07-25T08:00:00Z
 taskana.jobs.cleanup.minimumAge=P14D
+taskana.german.holidays.enabled=true
```

Comment:
But didn't you changed this to be the default? Why do we have to add it to all properties now?

Reply:
You requested to make this configurable. Therefore, I introduced configuration property taskana.german.holidays.enabled, which is a boolean. Usually the default value for a boolean property is false. Therefore, I didn't want to set this property to true if it is not specified. But if you say, I should, then I'll do this.